examples/testapp/src/pages/auto-sub-account/index.page.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  Box,
+  Button,
+  Container,
+  FormControl,
+  FormLabel,
+  Radio,
+  RadioGroup,
+  Stack,
+  VStack,
+} from '@chakra-ui/react';
+import { useState } from 'react';
+import { baseSepolia } from 'viem/chains';
+import { useConfig } from '../../context/ConfigContextProvider';
+import { useEIP1193Provider } from '../../context/EIP1193ProviderContextProvider';
+
+export default function AutoSubAccount() {
+  const [accounts, setAccounts] = useState<string[]>([]);
+  const [lastResult, setLastResult] = useState<string>();
+  const { subAccountsConfig, setSubAccountsConfig } = useConfig();
+  const { provider } = useEIP1193Provider();
+
+  const handleRequestAccounts = async () => {
+    if (!provider) return;
+
+    try {
+      const response = await provider.request({
```

Comment:
seeing this error when testing with auto sub accounts enabled

![image](https://github.com/user-attachments/assets/9a55d983-234b-4f98-b319-6f4ac28a639c)

Reply:
second attempt succeeds